<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janela_login</class>
 <widget class="QMainWindow" name="janela_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/conta.png</normaloff>:/icons/conta.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>16</y>
      <width>689</width>
      <height>321</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/icons/d0267b4d9d1d40dab9ec5eac4fedc3f8.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="barra_progresso">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>327</y>
      <width>649</width>
      <height>33</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
	border-radius: 14px;
	background-color:rgb(223, 201, 245)
}

QProgressBar::chunk{
	border-radius: 14px;
	background-color:rgb(126, 14, 237)

}
</string>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
